verify/src/metadata_schema.json

```diff
@@ -123,6 +124,28 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
+          "type": "string"
+        },
+        "weather_model": {
+          "type": "string",
+          "enum": [
+            "None",
+            "ERA5",
+            "ERA5T",
+            "GMAO",
+            "HRES",
+            "HRRR"
+          ]
+        },
+        "temporal_baseline_days": {
+          "type": "integer",
+          "minimum": 1
+        },
+        "frame_number": {
+          "type": "integer",
+          "minimum": 1
```

Comment:
I'm not sure if `frame_number` will be zero indexed. I expect there's a valid `maximum` as well but I'm fine moving ahead without that right now.